### ISSUE TITLE
(79, -4) portal-puzzle

### ISSUE THREAD

**Issue by sdilauro** [decentraland/godot-explorer#295]
- [ ] When the character enters a teleport Godot shows 'loading-screen' cutting the experience.

**Comment by kuruk-mm** [comment]
I was not able to reproduce it

https://github.com/decentraland/godot-explorer/assets/12563266/cfb17c43-7ee8-4d90-8813-e10b1ba632b7

**Comment by sdilauro** [comment]
Yes, idk why the day I created this problem it was not working properly. Today it works as you have posted in the video.
